pandas/core/indexes/accessors.py

```diff
@@ -472,6 +559,8 @@ def __new__(cls, data: Series):
                 index=orig.index,
             )
 
+        if isinstance(data.dtype, ArrowDtype) and data.dtype.kind in "mM":
```

Comment:
is it "mM" or just "M"?

Reply:
Thanks yeah this should be "M" until `pa.duration` is supported